packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -2240,16 +2252,14 @@ export class CrossChainMessenger {
      * @param message Message to generate the finalization transaction for.
      * @param opts Additional options.
      * @param opts.overrides Optional transaction overrides.
+     * @param messageIndex The index of the message, if multiple exist from multicall
      * @returns Gas estimate for the transaction.
      */
     finalizeMessage: async (
       message: MessageLike,
       opts?: {
         overrides?: CallOverrides
       },
-      /**
-       * The index of the withdrawal if multiple are made with multicall
```

Comment:
I took the liberty to move these up into `param`, can revert if not wanted

Reply:
In future will get rid of jsdoc comments but this is more consistent with now so I think this is a nice change